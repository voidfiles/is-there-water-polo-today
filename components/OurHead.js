import Head from "next/head";

export default function OurHead() {
  const baseURL =
    process.env.VERCEL_ENV === "production"
      ? "https://istherewaterpolo.today"
      : "";
  const twitterCard = baseURL + "/twitter_card.jpg";
  return (
    <Head>
      <title>Is there water polo today?</title>
      <meta
        property="og:title"
        content="Is there water polo today?"
        key="title"
      />
      <meta name="twitter:card" content="summary" />
      <meta name="twitter:site" content="@waterpolotoday" />
      <meta name="twitter:title" content="Is there water polo today?" />
      <meta
        name="twitter:description"
        content="A list of all the waterpolo games happening soon."
      />
      <meta name="twitter:image" content={twitterCard} />
    </Head>
  );
}
