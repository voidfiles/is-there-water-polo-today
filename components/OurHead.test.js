import React from "react";
import renderer from "react-test-renderer";

import OurHead from "./OurHead";

it("renders a simple head", () => {
  const tree = renderer.create(<OurHead />).toJSON();
  expect(tree).toMatchSnapshot();
});
