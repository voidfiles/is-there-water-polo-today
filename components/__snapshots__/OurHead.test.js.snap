// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a simple head 1`] = `null`;
